src/javascript/detectors/integer_overflow/integer_overflow.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=integer-overflow@v1.0 defects=1}
+function integerOverflowNoncompliant() {
+    // Noncompliant: big int is assigned to a variable.
+    var max = 1e21
```

Comment:
Is this enough code for customers to understand the issue? @nrosner

Reply:
Changed the example.